en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-alerts.md

```diff
@@ -1,2 +1,13 @@
-!!! Note
-    Content to be added. WIP.
\ No newline at end of file
+# Alerts Page
+This page illustrates information related to business alerts issued by the system for your currently selected
```

Comment:
Can we explain, 'system' and 'whole team'?

Suggestion:
"This page shows the information related to business alerts issued by the system for your currently selected
 environment and organization. You can use this page as a health monitoring dashboard and make it visible to 
the whole team. You can use this page to drill down on each alert and discover possible anomalies or errors in your published APIs. 

See the topics given below to get details of the available widgets and understand how to use them to best effect."

Reply:
fixed with cc3b796